### PR TITLE
adding aws-route53-play

### PR DESCRIPTION
**Pull request by rayashworth**

### CODE REVIEW

File: ansible/aws-route53-play/readme.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+edit **aws-route53-vars.yml** see file for details
+
+add aws route53 ansible module
+```
+# ansible-galaxy collection install -r requirements.yml
+```
+
+## add AWS record
+```
+# ansible-playbook -i inventory aws-route53-play -e route_task="add"
```
Comment (wkrapohl) [inline ansible/aws-route53-play/readme.md:48]: Change 
ansible-playbook -i inventory aws-route53-play -e route_task="add" 

Toansible-playbook -i inventory aws-route53-play.yml -e route_task="add"

**Review by wkrapohl — CHANGES_REQUESTED** [review]
One readme update in play. Otherwise looks like same code I've been using.

File: ansible/aws-route53-play/readme.md (hunk 1 above)
Comment (rayashworth) [reply ansible/aws-route53-play/readme.md]: done

**Review by wkrapohl — APPROVED** [review]
Good to go

**Pull request merged**